tests/test_llm_extractor.py

```
@@ -1,139 +1,48 @@
+    # 2. Definir un HTML de ejemplo y el esquema de extracción
+    sample_html = "<html><body><h1>Test Product</h1><p>Price: $99.99</p></body></html>"
+    extraction_schema = MockProduct
+
+    # 3. Llamar al método a probar
+    result = llm_extractor.extract(sample_html, extraction_schema)
+
+    # 4. Verificar que el cliente fue llamado correctamente
+    mock_openai_client.chat.completions.create.assert_called_once()
+    call_args, call_kwargs = mock_openai_client.chat.completions.create.call_args
+    # Verificamos que el HTML de ejemplo está en el prompt enviado al LLM
+    assert sample_html in str(call_kwargs['messages'])
```

Ruff (0.12.2)

Use of `assert` detected

(S101)